client/style.css

```diff
@@ -264,27 +265,38 @@ h1{
 .ig-thumbnail-inner-container {
   position: relative;
   overflow: hidden;
-  height: 100%;
+  height: 96%;
```

Comment:
Assuming this number is tied to the shims that surround it?

Reply:
Correct.